README.md

```diff
@@ -1028,13 +1030,15 @@ maintenance:
 
 
 ### Security
-| Parameter                        | Description                  | Default       |
-|:---------------------------------|:-----------------------------|:--------------|
-| `security`                       | Security configuration       | `{}`          |
-| `security.basic`                 | HTTP Basic configuration     | `{}`          |
-| `security.oidc`                  | OpenID Connect configuration | `{}`          |
-
-#### Basic
+| Parameter        | Description                                    | Default |
+|:-----------------|:-----------------------------------------------|:--------|
+| `security`       | Security configuration                         | `{}`    |
+| `security.basic` | HTTP Basic configuration                       | `{}`    |
+| `security.oidc`  | OpenID Connect configuration                   | `{}`    |
+| `web.cert_file`  | Public certificate file for TLS in PEM format. | ``      |
+| `web.key_file`   | Private key file for TLS in PEM format.        | ``      |
```

Comment:
These two don't belong there. Only configuration parameters from the `security` configuration should be there.